meringue-maven-plugin/src/main/java/edu/neu/ccs/prl/meringue/ReportFormat.java

```diff
@@ -0,0 +1,33 @@
+package edu.neu.ccs.prl.meringue;
+
+import org.jacoco.report.*;
+import org.jacoco.report.html.HTMLFormatter;
+import org.jacoco.report.xml.XMLFormatter;
+import org.jacoco.report.csv.CSVFormatter;
+
+import java.io.*;
```

Comment:
Maybe expand these; I've not seen `import java....*` in a while.

Reply:
OK